pytrip/ctx.py

```diff
@@ -102,7 +102,7 @@ def create_dicom(self):
 
         ds.KVP = ''  # KVP tag 0x0018, 0x0060
 
-        ds.AcquisitionNumber = ''  # AcquisitionNumber tag 0x0020, 0x0012
+        ds.AcquisitionNumber = 1  # AcquisitionNumber tag 0x0020, 0x0012
```

Comment:
tested it, '1' is also converted properly. Agree to stay close to the value representations.